source/auth/tests/legacy/connection-string.json

```diff
@@ -80,7 +80,7 @@
     },
     {
       "description": "should accept generic mechanism property (GSSAPI)",
-      "uri": "mongodb://user%40DOMAIN.COM@localhost/?authMechanism=GSSAPI&authMechanismProperties=SERVICE_NAME:other,CANONICALIZE_HOST_NAME:forward,SERVICE_HOST:example.com",
+      "uri": "mongodb://user%40DOMAIN.COM@localhost/?authMechanism=GSSAPI&authMechanismProperties=SERVICE_NAME:other,CANONICALIZE_HOST_NAME:true",
```

Comment:
is this correct change?

Reply:
This is basically reverting DRIVERS-1803 which only the Node driver implemented because it was needed for mongosh. I don't know if any of the other drivers will actually implement that so not sure what to do here.

Reply:
This change is unrelated to oidc, I think we can leave it as it is in the master

Reply:
Fixed

Reply:
Thanks!